.github/workflows/assets.yml

```diff
@@ -101,16 +96,15 @@ jobs:
           result-encoding: string
           script: |
             console.log(context)
-            tag = '${{ env.TAG }}'
 
             // first create a release -- it is OK if that fails,
             // since it means the release is already there
             try {
               const raw = (await github.repos.createRelease({
                 owner: context.repo.owner,
                 repo: context.repo.repo,
-                tag_name: tag,
-                name: 'Release ' + tag,
+                tag_name: TAG,
```

Comment:
Based on the errors we saw in 11.0 I suspect this and the two cases below needs to be ${TAG} (but I don't know the syntax for sure).
Or, since we will be using the assets.yml@master, we could delete this file or leave it unchanged in 10.4.

Reply:
Now that you have the correct syntax in master can you make this file match master. Thanks.